### PR TITLE
Replace x-common with problem-specifications

### PR DESCRIPTION
**Pull request by mienaikage**

### CODE REVIEW

File: app/views/contribute/canonical_data.erb [hunk 1]
```diff
@@ -56,7 +56,7 @@
             </p>
 
             <p>
-              The process for extracting canonical data is described in the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">Contributing Guide</a>.
+              The process for extracting canonical data is described in the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">Contributing Guide</a>.
```
Comment (kytrinyx) [inline app/views/contribute/canonical_data.erb]: This documentation now lives in https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/improving-consistency-across-tracks.md

File: app/views/contribute/canonical_data.erb [hunk 2]
```diff
@@ -85,11 +85,11 @@
...
+          This gets implemented into a file called <code>canonical-data.json</code>. The file lives in <a href="https://github.com/exercism/problem-specifications">exercism/problem-specifications</a> repository, alongside the other common data for the problem.</p>
 
           <p>So far we've extracted canonical data for <%= canonical_problems_count %> of the <%= active_problems_count %> problems. Just <%= problems.count %> more to go!</p>
 
-          <p>Want to help out? Pick an exercise from the list in the sidebar, and check out the documentation in <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">the contributing guide</a>.</p>
+          <p>Want to help out? Pick an exercise from the list in the sidebar, and check out the documentation in <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#improving-consistency-by-extracting-shared-test-data">the contributing guide</a>.</p>
```
Comment (kytrinyx) [inline app/views/contribute/canonical_data.erb]: This is the same as above.

File: app/views/contribute/index.erb [hunk 3]
```diff
@@ -115,15 +115,15 @@
...
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#beta-testing-a-language-track">beta testing</a> in the Contributing Guide.
           </p>
 
           <p>
             Didn't find a language? We're always looking to increase our supported language tracks. <a href="mailto:kytrinyx@exercism.io">Email Katrina</a>, she'll be glad to help.
           </p>
 
            <p>
-            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#starting-a-new-track">starting a new track</a> in the Contributing Guide.
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#starting-a-new-track">starting a new track</a> in the Contributing Guide.
```
Comment (kytrinyx) [inline app/views/contribute/index.erb]: Let's link to this instead: https://github.com/exercism/request-new-language-track/issues/new

File: app/views/contribute/index.erb [hunk 4]
```diff
@@ -135,11 +135,11 @@
           <h2>Make Up Exercises</h2>
 
           <p>
-            We're always looking to add new exercises. Have an idea? Suggest it in the <a href="https://github.com/exercism/x-common">metadata repository</a>.
+            We're always looking to add new exercises. Have an idea? Suggest it in the <a href="https://github.com/exercism/problem-specifications">metadata repository</a>.
           </p>
 
            <p>
-            Check out the section about <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">implementing a new exercise</a> in the Contributing Guide.
+            Check out the section about <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">implementing a new exercise</a> in the Contributing Guide.
```
Comment (kytrinyx) [inline app/views/contribute/index.erb]: This lives here: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/implementing-a-completely-new-exercise.md

File: app/views/contribute/index.erb [hunk 5]
```diff
@@ -157,7 +157,7 @@
           </p>
 
           <p>
-            Check out the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">detailed guide</a> if you want to give it a try!
+            Check out the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">detailed guide</a> if you want to give it a try!
```
Comment (kytrinyx) [inline app/views/contribute/index.erb]: This got rewritten here: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/porting-an-exercise.md

File: app/views/languages/_contribute.erb [hunk 6]
```diff
@@ -19,5 +19,5 @@ along with links to all the existing implementations.
 </p>
 
 <p>
-If you have an idea for a completely new exercise, you can follow <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">this guide</a> to add it to the site.
+If you have an idea for a completely new exercise, you can follow <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">this guide</a> to add it to the site.
```
Comment (kytrinyx) [inline app/views/languages/_contribute.erb]: As above

File: app/views/languages/_todo.erb [hunk 7]
```diff
@@ -13,13 +13,13 @@
 
 <p>The language icons link to each existing implementation of the problem.</p>
 <p>
-Once you've picked an exercise, follow the instructions in the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track" target="_blank">Contributing Guide</a> and submit your pull request to the <a href="<%= track.repository %>" target="_blank"><%= track.language %> track repository</a> on GitHub.
+Once you've picked an exercise, follow the instructions in the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track" target="_blank">Contributing Guide</a> and submit your pull request to the <a href="<%= track.repository %>" target="_blank"><%= track.language %> track repository</a> on GitHub.
```
Comment (kytrinyx) [inline app/views/languages/_todo.erb]: As above.

File: app/views/languages/_todo.erb [hunk 8]
```diff
@@ -13,13 +13,13 @@
...
 </p>
 
 <% todos = Trackler.todos[track.id] %>
 <% if todos.count.zero? %>
   <div class="col-md-12 jumbotron">
-    <p>All exercises are implemented in <%= track.language %>, you will need to <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">invent a new exercise.</a></p>
+    <p>All exercises are implemented in <%= track.language %>, you will need to <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">invent a new exercise.</a></p>
```
Comment (kytrinyx) [inline app/views/languages/_todo.erb]: As above.

File: app/views/languages/contribute.erb [hunk 9]
```diff
@@ -20,13 +20,13 @@
 
         <p>The language icons link to each existing implementation of the problem.</p>
         <p>
-        Once you've picked an exercise, follow the instructions in the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track" target="_blank">Contributing Guide</a> and submit your pull request to the <a href="<%= exercises.repository %>" target="_blank"><%= exercises.language %> track repository</a> on GitHub.
+        Once you've picked an exercise, follow the instructions in the <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track" target="_blank">Contributing Guide</a> and submit your pull request to the <a href="<%= exercises.repository %>" target="_blank"><%= exercises.language %> track repository</a> on GitHub.
```
Comment (kytrinyx) [inline app/views/languages/contribute.erb]: As above.

File: app/views/languages/contribute.erb [hunk 10]
```diff
@@ -20,13 +20,13 @@
...
         </p>
       </div>
     </div>
     <% if exercises.count.zero? %>
       <div class="col-md-12 jumbotron">
-        <p>All exercises are implemented in <%= exercises.language %>, you will need to <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">invent a new exercise.</a></p>
+        <p>All exercises are implemented in <%= exercises.language %>, you will need to <a href="https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">invent a new exercise.</a></p>
```
Comment (kytrinyx) [inline app/views/languages/contribute.erb]: As above.

File: app/views/languages/not_found.erb [hunk 11]
```diff
@@ -11,7 +11,7 @@
 
       <p>
         If you'd like to help implement it, then check out the
-        <a href="https://github.com/exercism/x-common/pull/125">Contributing Guide</a>, which
+        <a href="https://github.com/exercism/problem-specifications/pull/125">Contributing Guide</a>, which
```
Comment (kytrinyx) [inline app/views/languages/not_found.erb]: This Let's link to this instead: https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md

**Comment by mienaikage** [comment]
Replaced URLs are in https://github.com/exercism/exercism.io/pull/3567

**Comment by kotp** [comment]
Now needs a rebase, I think.

**Comment by kytrinyx** [comment]
@mienaikage Would you rebase this onto the most recent master?

**Comment by kytrinyx** [comment]
@kotp Jinx :)

**Pull request merged**